apps/mobile/src/components/PostCard/PostBodyProposal/PostBodyProposal.js

```diff
@@ -77,8 +77,12 @@ export default function PostBodyProposal ({
   id
 }) {
   const { t } = useTranslation()
-  const dispatch = useDispatch()
 
+  const [{ fetching: addProposalVoteMutationFetching }, addProposalVote] = useMutation(addProposalVoteMutation)
+  const [{ fetching: removeProposalVoteMutationFetching }, removeProposalVote] = useMutation(removeProposalVoteMutation)
+  const [{ fetching: swapProposalVoteMutationFetching }, swapProposalVote] = useMutation(swapProposalVoteMutation)
+
+  const isVoting = useMemo(() => addProposalVoteMutationFetching && addProposalVoteMutationFetching && addProposalVoteMutationFetching, [addProposalVoteMutationFetching, addProposalVoteMutationFetching, addProposalVoteMutationFetching])
```

Comment:
Sorry, my bad in my too quick notes earlier. This should simply be:

const isVoting = addProposalVoteFetching && removeProposalVoteFetching && swapProposalVoteFetching

Note the renames and reflect them starting with the `fetching: addProposalVoteMutationFetching` part, and no need for `useMemo` here.